projects/js-packages/publicize-components/src/components/post-publish-review-prompt/index.js

```diff
@@ -8,7 +8,7 @@ import { usePostStartedPublishing } from '../../hooks/use-saving-post';
 import useSocialMediaConnections from '../../hooks/use-social-media-connections';
 import ReviewPrompt from '../review-prompt';
 
-const PostPublishReviewPropmpt = () => {
```

Comment:
Funny typo that lived for a long type because of default exports 👀

Reply:
Yes, that is one of the reasons I hate default exports and always prefer to use named exports.